index.html

```diff
@@ -26,7 +26,7 @@
            </nav>
     </header>
     <main class="main">
-        <div class="container boss d-flex align-items-center">
+        <div class="container">
             <img class="promo-img" src="img/promo-negr.png" alt="Negr">
             <div class="flex-promo">
                 <div class="promo-text">
```

Comment:
вот сюда засунь твой список ниже, который с классом `flex-lis`. Потом просто флексами сделай и гапами в этом классе
```suggestion
                <div class="promo-text">
                    <p>Listen on</p>
                    <nav class="flex-png">
                        <ul class="flex-png">
                            <li><a href="#"><img src="img/SpotIfy.png" alt="spot"></a></li>
                            <li><a href="#"><img src="img/Pocket_casts.png" alt="now"></a></li>
                            <li><a href="#"><img src="img/soundcloud.png" alt="listen"></a></li>
                            <li><a href="#"><img src="img/podcast.png" alt="listen"></a></li>
                            <li><a href="#"><img src="img/x-proga.png" alt="listen"></a></li>
                        </ul>
                    </nav>
           </div>
```